types/phoenix_live_view/view.d.ts

```diff
@@ -0,0 +1,140 @@
+/// <reference types="node" />
+
+import Rendered from "./rendered";
+
+export function prependFormDataKey(key: any, prefix: any): any;
+export default class View {
+  static closestView(el: any): any;
+  constructor(el: any, liveSocket: any, parentView: any, flash: any, liveReferer: any);
+  isDead: boolean;
+  liveSocket: any;
+  flash: any;
+  parent: any;
+  root: any;
+  el: any;
+  id: any;
+  ref: number;
+  lastAckRef: any;
+  childJoins: number;
+  loaderTimer: NodeJS.Timeout | null;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/setTimeout#return_value documents the return value as a `positive integer`. The primitive type in Node is also a number: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/697a818a776f93f45756b3b17a0d277cdc8f0861/types/node/timers.d.ts#L82-L125

I think we could keep this package independent of `types/node`, but won't insist on it. You'd already done a awesome job getting this up-to-date, thanks!

Reply:
Point taken! I changed the types to `number | null` and removed the dependency.